lib/yoti/data_type/attribute.rb

```diff
@@ -1,25 +1,25 @@
 module Yoti
   # Encapsulates profile attribute
   class Attribute
-    FAMILY_NAME = 'family_name';
-    GIVEN_NAMES = 'given_names';
-    FULL_NAME = 'full_name';
-    DATE_OF_BIRTH = 'date_of_birth';
-    GENDER = 'gender';
-    NATIONALITY = 'nationality';
-    PHONE_NUMBER = 'phone_number';
-    SELFIE = 'selfie';
-    EMAIL_ADDRESS = 'email_address';
-    POSTAL_ADDRESS = 'postal_address';
-    STRUCTURED_POSTAL_ADDRESS = 'structured_postal_address';
+    FAMILY_NAME = 'family_name'.freeze
```

Comment:
I'm considering it might be best to leave these `.freeze`s until the major release. If we're being pedantic, it's a breaking change, albeit it's not a way clients should ever be using the SDK. @bucky-boy what do you think?

Reply:
@echarrod I've reverted this in 7af42f3 as well as excluding the _Naming/PredicateName_ rule so that we don't need to rename `Yoti::AgeProcessor.is_age_verification()`